sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/WebPubSubServiceClient.java

```diff
@@ -353,9 +357,14 @@ public Response<Void> addConnectionToGroupWithResponse(String group, String conn
      * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Response<Void> addConnectionsToGroupsWithResponse(String hub, BinaryData groupsToAdd,
+    public Response<Void> addConnectionsToGroupsWithResponse(String hub, List<String> groups, String filter,
```

Comment:
Should this API be `void addConnectionsToGroups(String hub, List<String> groups, String filter)`?

Reply:
Just checked with Liangying, for this we'll keep the current method signature to follow the current convention (other methods also follow this pattern as well)
cc @vicancy

Reply:
Please see this API and their differences. Generally speaking, the `WithResponse` API is not recommended to be updated. You can hide it, but not recommend to do such modification on it.
https://github.com/Azure/azure-sdk-for-java/blob/deef3da0ab9b030bcf28c8b941f45764380b6e78/sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/WebPubSubServiceClient.java#L121-L145

Loop @srikanta

Reply:
okay understood, I'll undo the changes on the WithResponse API method and wrap it with a void method then

Reply:
@weidongxu-microsoft just pushed new changes. Please let me know if anything else needs to improve

Reply:
`addConnectionsToGroups`

no `requestOptions` param

Same apply to the Async client.

Reply:
@weidongxu-microsoft done. Can you please recheck?

Reply:
The public method is `addConnectionsToGroups`. No `WithResponse`.